README.md

```diff
@@ -5,8 +5,22 @@ At this moment the following types are available:
 
 | Package | Python Class | Description |
 | ------------- | ------------- | ------------- |
-| google.events.cloud.pubsub.v1 | MessagePublishedData | A message that is published by publishers and consumed by subscribers. |
-| google.events.cloud.audit.v1 | LogEntryData | This event is triggered when a new audit log entry is written. |
+| google.events.cloud.scheduler.v1 | SchedulerJobData | Scheduler job data. |
+| google.events.cloud.pubsub.v1 | MessagePublishedData | The data received in an event when a message is published to a topic. |
+| google.events.firebase.database.v1 | ReferenceEventData | The data within all Firebase Real Time Database reference events. |
+| google.events.firebase.remoteconfig.v1 | RemoteConfigEventData | The data within all Firebase Remote Config events. |
+| google.events.firebase.auth.v1 | AuthEventData | The data within all Firebase Auth events |
+| google.events.cloud.storage.v1 | StorageObjectData | An object within Google Cloud Storage. |
+| google.events.cloud.firestore.v1 | DocumentEventData | The data within all Firestore document events. |
+| google.events.firebase.analytics.v1 | AnalyticsLogData | The data within Firebase Analytics log events. |
+| google.events.cloud.audit.v1 | LogEntryData | Generic log entry, used as a wrapper for Cloud Audit Logs in events.
+ This is copied from
```

Comment:
I don't think this table has the right contents here.

Reply:
Do you mean the line breaks? I was not aware that we have line breaks in the description field now; will update the PR.

Reply:
The description is now formatted correctly.